websites/D/Discords.com/presence.ts

```diff
@@ -0,0 +1,175 @@
+      presenceData.state = `📖 Page ${page}`;
+      presenceData.buttons = [
+        {
+          label: "View Tag",
+          url: document.location.href
+        }
+      ];
+    } else if (document.location.pathname.includes("/servers/tags/")) {
+      const page = document.querySelector("li.page-item.active")?.textContent,
+      tag = document.querySelector("h1.text-center")?.textContent.replace("Discord Servers", " ");
+      presenceData.details = `Viewing 🔑 ${tag || "Nothing"} keyword`;
+      presenceData.state = `📖 Page ${page}`;
+      presenceData.buttons = [
+        {
+          label: "View Keyword",
+          url: document.location.href
+        }
+      ];
+    } else if (document.location.pathname === "/servers/top-100") {
+      const page = document.querySelector("li.page-item.active")?.textContent;
+      presenceData.details = "Viewing 🏅 top-100 servers";
+      presenceData.state = `📖 Page ${page}`;
+      presenceData.buttons = [
+        {
+          label: "View Top-100",
+          url: document.location.href
+        }
+      ];
+    } else if (document.location.pathname.includes("/join")) {
+      const server_name_join = document.querySelector("h2.mt-4")?.textContent;
+      presenceData.details = `Joining 🚦 ${server_name_join || "Nothing"} `;
+      presenceData.buttons = [
+        {
+          label: `Join ${server_name_join || "Nothing"}`,
+          url: document.location.href
+        }
+      ];
+    } else if (document.location.pathname.includes("/upvote")) {
+      const server_name_upvote = document.querySelector("h1.JoinUpvote_clickable-server-name__3l6Es")?.textContent;
+      presenceData.details = `Upvoting 🗳️ ${server_name_upvote || "Nothing"} `;
+      presenceData.buttons = [
+        {
+          label: `Upvote ${server_name_upvote || "Nothing"}`,
+          url: document.location.href
+        }
+      ];
+    } else if (document.location.pathname.includes("/servers/")) {
+      const join = document.querySelectorAll("a.btn")[1]?.getAttribute("href"),
+      upvote = document.querySelectorAll("a.btn")[2]?.getAttribute("href"),
+      membersonline = document.querySelectorAll("span.mr-2")[0]?.textContent.replace(`${document.querySelectorAll("span.mr-2")[0]?.textContent}`, `${"👥" + document.querySelectorAll("span.mr-2")[0]?.textContent}`),
+      members = document.querySelectorAll("span.mr-2")[1]?.textContent.replace(`${document.querySelectorAll("span.mr-2")[1]?.textContent}`, `${"🎁" + document.querySelectorAll("span.mr-2")[1]?.textContent}`),
+      boosts = document.querySelectorAll("span.mr-2")[2]?.textContent.replace(`${document.querySelectorAll("span.mr-2")[2]?.textContent}`, `${"💎" + document.querySelectorAll("span.mr-2")[2]?.textContent}`),
+      server = document.querySelector("h1.servernameh1")?.textContent.replace("PREMIUM", " ");
+      presenceData.details = `Viewing 🎨 ${server}`;
+      presenceData.state = `${membersonline || "N/A"}, ${members || "N/A"}, ${boosts || "No boosts"}`;
```

Comment:
N/A, N/A, No boots
it doesn't look good xD

Reply:
Should I just put "0"? 0 members online, 0 members, 0 boosts?